### PR TITLE
Add GIFs and project dependencies

### PR DESCRIPTION
**Pull request by ritwik4690**
Added gifs to ReadME for explaining how to use functionality. Also added project dependencies

### CODE REVIEW

**Review by rishikesh-yelne — APPROVED** [review]

**Review by rishikesh-yelne — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -152,9 +152,14 @@ dmypy.json
...
+
+
 # PyCharm
 #  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-.idea/
\ No newline at end of file
+.idea/.DS_Store
```
Comment (mithildani) [inline .gitignore:165]: yeh galat hai
i need all files inside .idea to be ignored

File: README.md [hunk 2]
```diff
@@ -7,44 +7,18 @@ SlackPoint!
 [![build](https://github.com/mithildani/se22-hw1-grp14/actions/workflows/test.yml/badge.svg)](https://github.com/mithildani/se22-hw1-grp14/actions)
 [![GitHub top language](https://img.shields.io/github/languages/top/mithildani/se22-hw1-grp14)](https://docs.python.org/3/)
 [![GitHub last commit](https://img.shields.io/github/last-commit/mithildani/se22-hw1-grp14)](https://github.com/mithildani/se22-hw1-grp14/commits/main)
-[![codecov](https://codecov.io/gh/nehakale8/slackpoint/branch/main/graph/badge.svg?token=1H92SAVB5S)](https://codecov.io/gh/nehakale8/slackpoint)
```
Comment (mithildani) [inline README.md:10]: Why is codecov badge removed?

**Review by mithildani — DISMISSED** [review]

**Comment by mithildani** [comment]
Need to add a future scope section
where we can show how we can extend our application

things I can think of:
1. assign users to tasks while creating each task
2. Allow a command to reassign users
3. Allow a command to edit existing task
4. Progress of a task is currently binary. Improve existing command to allow a percentage progress improvement
5. Deploy the service on cloud
6. Improve code coverage
7. UI/UX: Improve leaderboard command response to show gifs/graphs to further make the leaderboard more attractive and gamify it
8.

File: README.md (hunk 2 above)
Comment (ritwik4690) [reply README.md:10]: my bad

**Review by rishikesh-yelne — APPROVED** [review]

**Pull request merged**